### DISCUSSION TITLE
Get Error: Cannot find a file ending with ".routes.ts or -routing.module.ts" to add the route. when generating a new page.

### DISCUSSION THREAD

**Discussion by suprcodr** [cyrilletuzi/vscode-angular-schematics#918] [Q&A] [answered]
Hi,

# Checks before posting an issue

- [x ] I have read _all_ the documentation included in the extension itself as a walkthrough, including the [troubleshooting guide](https://github.com/cyrilletuzi/vscode-angular-schematics/blob/main/TROUBLESHOOTING.md) (from the VS Code menu: View > Command Palette >search "Documentation" > choose "Angular Schematics: Documentation and tutorial")
- [x ] I have checked in already:
  - [opened issues](https://github.com/cyrilletuzi/vscode-angular-schematics/issues)
  - [closed issues](https://github.com/cyrilletuzi/vscode-angular-schematics/issues?q=is%3Aissue+is%3Aclosed)
  - [questions answered](https://github.com/cyrilletuzi/vscode-angular-schematics/discussions/categories/q-a) in the new "Discussions" tab
- [x ] My project meets the requirements indicated in README
- [x ] My workspace is in [trusted mode](https://code.visualstudio.com/docs/editor/workspace-trust), not restricted mode
- [x ] I looked at the extension's logs ("Angular Schematics" Output channel, second tab left to the Terminal), and tried to fix my project's configuration accordingly
- [x ] I understand I am writing to a *human being*
- [x ] I will be involved in my own issue, by answering questions and informing if the issue is resolved

# Configuration

- VS Code version (only the last 2 versions are supported): 
- Extension version: 
- OS: 
- Are you using VS Code in normal mode or in a special mode (web version, WSL, SSH/remote, etc.): 

# Description of the issue
![image](https://github.com/cyrilletuzi/vscode-angular-schematics/assets/6827871/7a617e4a-f739-4f48-9d85-8fd6dfe03076)

I get the error: Cannot find a file ending with ".routes.ts or -routing.module.ts" to add the route. when generating a new page. When addind a new page. Am I doing something worng?

# Extension logs

```
Copy the result here
```

# How to reproduce the issue

**Comment by cyrilletuzi** [comment, marked as answer]
Hi @suprcodr,

Thank you for your subscription to the Pro edition. I have sent you an email to help you to get things right, it will be easier and faster.

(Also, moved this to discussions as it is not an issue)
